soc/x86/atom/Kconfig.defconfig

```diff
@@ -11,14 +11,6 @@ config SOC
 config SYS_CLOCK_HW_CYCLES_PER_SEC
 	default 25000000  if HPET_TIMER
 
-if BT_UART
-
-config UART_PIPE_ON_DEV_NAME
-	default "UART_1"
-	depends on UART_PIPE
```

Comment:
Do we need to add a `zephyr, chosen` node for this?

Reply:
I did that for `boards/arm/qemu_cortex_a9/qemu_cortex_a9.dts`, any others I should edit? I'm not familiar with `atom` though.

Reply:
It appears that `CONFIG_BT_UART` isn't used in code anymore, and was reworked to use `zephyr,bt-uart`.